tests/kafkatest/services/zookeeper.py

```diff
@@ -45,7 +45,7 @@ class ZookeeperService(KafkaPathResolverMixin, Service):
 
     # After 4.0, zookeeper service is removed from source code. Using LATEST_3_8 for compatibility test cases.
```

Comment:
nit.
LATEST_3_8 -> LATEST_3_9 in this comment.

Reply:
Thanks for pointing out. 😺 
Done